front/app/components/UI/FileRepositorySelectAndUpload/FileAttachmentDisplay.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+
+import {
```

Comment:
This is originally a duplicated FileDisplay component, then changed to handle File Attachments (and make the necessary call to GET the File).

It felt cleaner than trying to generalize the FileDisplay, and once we replace the old File handling system in the front office we can just use this component going forwards.